src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
```suggestion
        return Colors.values()[index].name();
```
![image](https://user-images.githubusercontent.com/81781479/193443138-51cc3fcd-58e5-40c0-aade-6ec1107c2192.png)
Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there. The mentor will not check the work until the checklist points are corrected